src/hyrdate.js

```diff
@@ -1,20 +1,30 @@
 class Hydration {
-  constructor(hydrationData) {
-    this.userId = hydrationData.userID,
-    this.date = hydrationData.date,
-    this.numOunces = hydrationData.numOunces
+  constructor(hydrationData, userID) {
+    this.hydrationData = hydrationData;
+    this.userID = userID;
   }
 
-  calculateAverageDailyConsumption() {
+  calculateTotalConsumption(id) {
+    let totalConsumption = this.hydrationData.reduce((acc, total) => {
+      if(total.userID === id) {
+        acc += total.numOunces;
+      }
+      return acc
+    }, 0)
+    return totalConsumption;
+      }
 
-  }
 
-  returnDailyConsumption() {
 
+  returnDailyConsumption(date) {
```

Comment:
Since this one is the specific consumption for a particular user on an identified date, it will need to parameters in the method as it will always be passing in two arguments, the users ID and the date.